src/init/AppRunner.ts

```diff
@@ -45,108 +48,70 @@ export class AppRunner {
    * @param args - Command line arguments.
    * @param stderr - Standard error stream.
    */
-  public runCli({
+  public async runCli({
```

Comment:
This should remain sync. Otherwise we may have uncaught promise rejections in `bin/server.js`.

Reply:
Okay. Will change it.

Reply:
It is now synchronous